pyproject.toml

```diff
@@ -9,6 +9,7 @@ typeCheckingMode = "strict"
     "tests/test_frontend_op_resolver.py",
     "tests/test_frontend_python_code_check.py",
     "tests/dialects/test_memref.py",
+    "xdsl/interpreters/experimental/wgpu.py",
```

Comment:
eclude means the errors will still be there if you open the file in your IDE, but not otherwise